dags/kombu_helper.py

```diff
@@ -0,0 +1,37 @@
+from kombu import Connection
+from kombu.simple import SimpleQueue
+from time import sleep
+import requests
+
+def check_queue(queue):
+    ret = requests.get("http://rabbitmq:15672/api/queues/%2f/{}".format(queue), auth=('guest', 'guest'))
+    if not ret.ok:
+        raise RuntimeError("Cannot connect to rabbitmq management interface")
+    queue_status = ret.json()
+    return queue_status
+
+
+def drain_messages(conn, queue):
+    with Connection(conn) as conn:
+        simple_queue = conn.SimpleQueue(queue)
+        num_drained = 0
+        while True:
+            try:
+                message = simple_queue.get_nowait()
+            except SimpleQueue.Empty:
+                status = check_queue(queue)
+                if "messages" not in status:
+                    sleep(30)
+                    continue
+                elif status["messages"] > 0:
+                    print(f'still {status["messages"]} messages left in the queue, sleep for 30s')
+                    sleep(30)
+                    continue
+                else:
+                    break
+            #print(f'message: {message.payload}')
+            message.ack()
+            num_drained += 1
+
+        simple_queue.close()
+        print(f"# tasks drained: {num_drained}")
```

Comment:
It seems this is very similar to the one used in slack bot, maybe we can merge the two and place the code under the `common` folder. No need to do it in this pr.